components/HomePage/12Press/EachPress.jsx

```diff
@@ -15,13 +17,25 @@ const EachPress = ({ press }) => {
 
   return (
     <Col lg={8} className="press-item">
-      <a href={link}>
-        <img src={`${process.env.NEXT_PUBLIC_API_URL}${imageUrl}`} alt={title} className="press-item-image" />
+      <a href={link} target="_blank" rel="noopener noreferrer">
+        <Image
+          src={`${process.env.NEXT_PUBLIC_API_URL}${imageUrl}`}
```

Comment:
The way we're referencing paths really needs to be more consistent and unified too. We have this way, getHostname() etc and it's not clear what the rationale is in each instance.